### PR TITLE
Implement the Settings page in the Svelte Web UI

### PR DESCRIPTION
**Pull request by rgsk**

### CODE REVIEW

**Comment by NIkhilbadveli** [comment]
It works well. Nice one.

**Review by Israel-Laguan — APPROVED** [review]

File: ui/src/routes/settings/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<script>
+  import Settings from "./Settings/Settings.svelte";
+
+
+</script>
+<div><Settings data/></div>
```
Comment (Israel-Laguan) [inline ui/src/routes/settings/+page.svelte:6]: Try to format this file

Comment (rgsk) [reply ui/src/routes/settings/+page.svelte]: done

**Comment by rgsk** [comment]
I have 1 issue, when I move Settings.svelte component in components folder, it get's ignored by git

**Review by Israel-Laguan — APPROVED** [review]

**Comment by rgsk** [comment]
@mufeedvh who will merge this?

**Comment by Israel-Laguan** [comment]
I think @123vivekr merge

**Comment by mufeedvh** [comment]
Thank you so much for implementing this, merging this now! ❤️ 🙌

**Pull request merged**